docs/reference/eql/eql.asciidoc

```diff
@@ -390,20 +390,15 @@ or event category field.
 
 By default, the EQL search API returns matching hits by timestamp. If two or
 more events share the same timestamp, {es} uses a tiebreaker field value to sort
-the events in ascending, lexicographic order.
+the events in ascending, lexicographic order. {es} orders events with no
```

Comment:
I would rephrase lexicographic to "their natural order". That's because tiebreakers are numeric and their lexicographic order might not match their order (0100 vs 20 -> "0100" vs "20").